apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.js

```diff
@@ -91,6 +92,10 @@ router.get('/:file_transfer_id', function (req, res, next) {
                 { uuid: editor.uuid, course_id: res.locals.course.id },
                 (err, result) => {
                   if (ERR(err, next)) return;
+                  flash(
+                    'success',
+                    'Question copied successfully. You are now viewing your copy of the question.'
```

Comment:
Loving the flash!

Reply:
Yes! IMO we could use a lot more of these now that we have the capability.